.github/workflows/action.yml

```diff
@@ -20,13 +20,6 @@ jobs:
           sed -i'.bak' \
               -e "s/const setupBeamVersion = '.*'/const setupBeamVersion = '${SHA}'/g" \
               dist/index.js
-          # it always starts by being 'undefined'
```

Comment:
Is this intentional?

Reply:
Yes. I mean it's not directly related to my initial intention, but it doesn't always start by being `undefined`, since once we change it, it goes to the `dist` and then it's not `undefined` any more 👍

Reply:
I ended up tweaking it more after I tried to do another pull request and run into "issues". (basically we need to update the `.js` that we usually do and then run `npm run build-dist` instead of only doing the change in the `dist` files)